notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>I think we should use the same definitions as in https://cloud.google.com/vertex-ai/docs/tabular-data/forecasting/train-model#training-methods. Please let me know if the definitions in the public doc are too short / missing key information. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Fixed.</p>

Reply:
<p>Nit: "the user won't need" -&gt; "you don't need"</p>

Reply:
<p>Fixed.</p>